test/functional/rpc_blockchain.py

```diff
@@ -255,6 +256,13 @@ def _test_getdeploymentinfo(self):
         # calling with an explicit hash works
         self.check_signalling_deploymentinfo_result(self.nodes[0].getdeploymentinfo(gbci207["bestblockhash"]), gbci207["blocks"], gbci207["bestblockhash"], "started")
 
+    def _test_y2106(self):
```

Comment:
You should clarify what this is testing.

Reply:
I think the code is simple and self-explanatory, no?

Reply:
Maybe, the function name suggests this is testing the 2106 bug. Could rename to e.g. `_test_y2106-1s` or something.

Reply:
Thanks, added info print